.github/release-drafter.yml

```diff
@@ -0,0 +1,20 @@
+name-template: v$NEXT_PATCH_VERSION
+tag-template: v$NEXT_PATCH_VERSION
+categories:
+  - title: 💡 New Features
+    label: feature
+  - title: 🔨 Breaking
+    label: breaking
+  - title: 🚀 Efficiency Improvement
+    label: efficiency
+  - title: 🐛 Bug Fixes
+    label: fix
+  - title: 📖 Documentation
+    label: doc
+  - title: 🧰 Maintenance
+    label: maintenance
```

Comment:
@StrikerRUS @jameslamb any better emojis for these items?

Reply:
@guolinke I think these ones are quite good! 👍

Reply:
I agree, these are good